.github/workflows/cleanup-previews.yml

```
@@ -172,14 +172,13 @@ jobs:
           </html>
           EOF
 
-      - name: Upload artifact
-        uses: actions/upload-pages-artifact@v3
+      - name: Deploy to GitHub Pages (Cleanup)
+        uses: peaceiris/actions-gh-pages@v3
```

actionlint (1.7.7)

the runner of "peaceiris/actions-gh-pages@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)